src/components/16-homepage/tile.njk

```diff
@@ -0,0 +1,33 @@
+<h3 class="margin-top-0">Basic Tile Example</h3>
+<div class="tile-wrapper">
```

Comment:
is `title-wrapper` part of the title component? if it is; adjust your classes so the parent container its the name of the component `sds-title`